.ci/azure/linux.yml

```diff
@@ -354,7 +354,7 @@ jobs:
     displayName: 'IE UT'
     continueOnError: false
 
-  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/cpuUnitTests --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-cpuUnitTests.xml
+  - script: $(RUN_PREFIX) $(INSTALL_TEST_DIR)/ov_cpu_unit_tests --gtest_output=xml:$(INSTALL_TEST_DIR)/TEST-ov_cpu_unit_tests.xml
     displayName: 'CPU UT'
```

Comment:
I propose to rename targets too

Reply:
Intel CPU Unit Tests for example

Reply:
Sorry, a little confused, changing displayName from 'CPU UT' to 'Intel CPU Unit Tests'? @ilyachur

Reply:
cmake target name: TARGET_NAME is ov_cpu_unit_tests.
https://github.com/xipingyan/frameworks.ai.openvino.ci.product-configs/blob/master/conf_dldt_test.py#L83 has provided name format.

Reply:
Yes I am talking about more human readable `displayName`

Reply:
Totally agree.
Changed cpu unit test name displayName from "CPU UT" to "Intel CPU Unit Tests".
For ov_cpu_func_tests's displayName: 'CPU FuncTests', I think it is readable, so don't change.